kit/Cargo.toml

```diff
@@ -15,6 +15,7 @@ icons = { path="../icons" }
 timeago = "0.4.0"
 humansize = "2.0.0"
 shared = { path = "../shared" }
+warp = { git = "https://github.com/Satellite-im/Warp", rev = "ed3b5de968fb623a7d76170a5d7dd3f3c79d3e1b"}
```

Comment:
Im not against this (since it would make it easier to convert from/to other fields than what I temporarily did for mapping identity status and platform), though i think the kit crate wasnt suppose to include warp.

Reply:
The UI Kit should be purely UI and not include any deps to warp.